client/zeth/core/prover_client.py

```diff
@@ -5,9 +5,11 @@
 # SPDX-License-Identifier: LGPL-3.0+
 
 from __future__ import annotations
-from .pairing import PairingParameters, pairing_parameters_from_proto
+from zeth.core.zksnark import IZKSnarkProvider, get_zksnark_provider, \
+    IVerificationKey, ExtendedProof
+from zeth.core.pairing import PairingParameters, pairing_parameters_from_proto
 from zeth.api.zeth_messages_pb2 import ProofInputs
-from zeth.api.snark_messages_pb2 import VerificationKey, ExtendedProof
+# from zeth.api.snark_messages_pb2 import VerificationKey, ExtendedProof
```

Comment:
Do we need (or is it handy) to keep this comment around? If not, let's remove it :)

Reply:
removed